Appsilon/shiny.telemetry
# Retrieve username from session

Issues:
- closes #73

## Changes description

* Gets username from `session$user` instead of the URL (documentation from [session object](https://shiny.rstudio.com/reference/shiny/latest/session.html))

## Testing

I've deployed a testing branch to Appsilon's connect server _(that should be removed after this PR is merged)_

Link to application: https://connect.appsilon.com/user-name-test-b2671b0120eb/

Steps:

1. Open a private window and access the link above
2. Log into connect.appsilon.com with valid credentials and access the link above
3. Open the deployed app logs [via link](https://connect.appsilon.com/connect/#/apps/6f362ec0-a9c1-4a74-9a69-b2671b0120eb/logs) and see something similar to the image below
   * _(only available to the team members that has access permissions)_

![image](https://user-images.githubusercontent.com/211358/231450449-8411a294-c24a-437f-9e32-52f60f987b6b.png)
